packages/mds-transactions-service/repository/entities/transaction-entity.ts

```diff
@@ -40,6 +40,7 @@ export class TransactionEntity extends IdentityColumn(RecordedColumn(class {}))
   device_id: TransactionEntityModel['device_id']
 
   @Column('bigint', { transformer: BigintTransformer })
+  @ForceType(Number)
```

Comment:
Are you going to disconnect the `TransactionEntityModel` from the `TransactionDomainModel` so that `@ForceType` isn't required?

Reply:
Yeah, I'm gonna do that. Will still keep `@ForceType` around in `mds-repository` though, could certainly be useful at some point.